pkg/app/mutate_app_test.go

```diff
@@ -236,11 +248,150 @@ func Test_MutateApp(t *testing.T) {
 			configMaps: []*corev1.ConfigMap{
 				newTestConfigMap("other-app-values", "eggs2"),
 			},
+			operation: v1beta1.Create,
+			expectedPatches: []mutator.PatchOperation{
+				mutator.PatchAdd("/metadata/annotations", map[string]string{}),
+			},
+		},
+		{
+			name: "case 6: set app-operator paused annotation and config-controller version label for CM app on CREATE",
```

Comment:
```suggestion
			name: "case 6: set app-operator paused annotation and config-controller version label for MC app on CREATE",
```

Shouldn't this be MC?